webdriver-spec.html

```diff
@@ -2010,6 +2010,11 @@
   <td>Specifies the <a>session script timeout</a>.
  </tr>
 </table>
+
+<p>When deserializing a <a>session timeouts configuration</a> other
+ keys and values are valid, but only those in the table above are
+ checked for correctness.
```

Comment:
The wording here is too loose. The algorithm from [Set Timeouts](https://w3c.github.io/webdriver/webdriver-spec.html#dfn-set-timeouts) should be moved here and defined as “The JSON deserialization of session timeouts configuration is…”. Then that can be reused in Set Timeouts and New Session.